docs: mention `isLoopFinished()` stop condition in context of agent loops (#14539)
vercel/ai: content/docs/07-reference/04-ai-sdk-workflow/01-workflow-agent.mdx

      name: 'stopWhen',
      type: 'StopCondition | StopCondition[]',
      isOptional: true,

>       description: 'Default stop condition for the agent loop. Per-stream values override this default. Use `isLoopFinished()` to let the agent run until all tool calls have completed, but pair it with a maximum step limit to avoid runaway loops. See https://ai-sdk.dev/v7/docs/reference/ai-sdk-core/loop-finished#isloopfinished.',

Author: ```suggestion
      description: 'Default stop condition for the agent loop. Per-stream values override this default. Use `isLoopFinished()` to let the agent run until all tool calls have completed, but beware of potential runaway loops. See https://ai-sdk.dev/v7/docs/reference/ai-sdk-core/loop-finished#isloopfinished.',
```

      name: 'stopWhen',
      type: 'StopCondition | StopCondition[]',
      isOptional: true,

>       description: 'Condition(s) for ending the agent loop. Use `isLoopFinished()` to let the agent run until all tool calls have completed, but pair it with `maxSteps` to avoid runaway loops. See https://ai-sdk.dev/v7/docs/reference/ai-sdk-core/loop-finished#isloopfinished.',

Author: ```suggestion
      description: 'Condition(s) for ending the agent loop. Use `isLoopFinished()` to let the agent run until all tool calls have completed, but beware of potential runaway loops. See https://ai-sdk.dev/v7/docs/reference/ai-sdk-core/loop-finished#isloopfinished.',
```
